.zenodo.json

```diff
@@ -1,54 +0,0 @@
-{
```

Comment:
wait, this is not possible zenodo.json is used to publish alpaka and requred for the webhook!
The author list in this file is requred ro auto generate the author list for the doi entry.

Reply:
No, when you have a `CITATION.cff`, zenodo will take this one and ignore the `.zenodo.json`. See here: https://github.com/citation-file-format/citation-file-format/issues/374#issuecomment-1061022832

Reply:
I was not aware that this works, that's nice.
We need to test the release workflow in another project first to avoid that we create a wrong release which is not removable in zenodoo.